rb/spec/integration/selenium/webdriver/chrome/service_spec.rb

```diff
@@ -37,6 +37,19 @@ module Chrome
         it 'can be started outside driver' do
           expect(service_manager.uri).to be_a(URI)
         end
+
+        it 'uses the path from the environment' do
+          ENV['SE_CHROMEDRIVER'] = DriverFinder.new(Options.new, described_class.new).driver_path
```

Comment:
This would leak into all subsequent specs - you need to clear `ENV` properly after the test is done.